seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -212,14 +225,15 @@ def test_create_project_from_workspace(self, mock_file_iter, mock_file_exist, mo
         self.assertEqual(response.reason_phrase, 'Project "{name}" for workspace "{namespace}/{name}" exists.'
                          .format(namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME))
 
-        # Test sending email exception
+        # Test saving ID file exception
         url = reverse(create_project_from_workspace, args=[TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME2])
-        mock_email.side_effect = Exception('Something wrong while sending email.')
+        mock_mv_file.side_effect = Exception('Something wrong while moving the ID file.')
         response = self.client.post(url, content_type='application/json', data=json.dumps(REQUEST_BODY))
         self.assertEqual(response.status_code, 200)
-        mock_api_logger.error.assert_called_with('AnVIL loading request email exception: Something wrong while sending email.', self.manager_user)
+        mock_api_logger.error.has_called_with('Uploading sample IDs to Google Storage failed. Errors: Something wrong while moving the ID file.',
```

Comment:
`has_called_with` is not a thing, this isn't testing anything

Reply:
Thank you. I must be more careful with this kind of mistake.